[mtt] destination is unreachable with UCX_TLS=tcp

Configuration
```
OMPI: 4.0.2a1
MOFED: MLNX_OFED_LINUX-4.5-1.0.1.0
Module: hpcx-gcc (2019-04-19)
Test module: none
Nodes: jazz x8 (ppn=28(x8), nodelist=jazz[04,06,08-09,13,21,24-25])
Job name: ucx-emulation-roce
```

MTT log: http://hpcweb.lab.mtl.com/hpc/mtr_scrap/users/mtt/scratch/shmem/20190420_022022_131250_20464_jazz04/html/test_stdout_lXPlxR.txt

Test works fine with `NP <= 160`. It also works with `NP == 224` without TCP enabled (device `mlx5_3:1`).

Cmd:
`oshrun -np 224 --display-map -mca btl self --bind-to core -mca oshmem_proc_group_cache_size 10000 -mca sshmem mmap -mca spml_ucx_heap_reg_nb 0 -mca coll '^hcoll' -mca coll_hcoll_enable 0 -mca spml ucx -mca pml ucx -x UCX_TLS=tcp -x UCX_NET_DEVICES=p2p2 --map-by node -x SHMEM_SYMMETRIC_HEAP_SIZE=128M /hpc/mtr_scrap/users/mtt/scratch/shmem/20190420_022022_131250_20464_jazz04/installs/Vmv3/tests/mpp_24/hpc_tests.git/mpp_bench_v1.0/bin/barrier.shmem 10`

Output:
```sh
[jazz04:185061] spml_ucx.c:264  Error: ucp_ep_create(proc=142/224) failed: Destination is unreachable
[jazz04:185061] spml_ucx.c:306  Error: add procs FAILED rc=-2
[jazz04:185059] spml_ucx.c:264  Error: ucp_ep_create(proc=139/224) failed: Destination is unreachable
[jazz04:185059] spml_ucx.c:306  Error: add procs FAILED rc=-2
[jazz04:185051] spml_ucx.c:264  Error: ucp_ep_create(proc=137/224) failed: Destination is unreachable
[jazz04:185051] spml_ucx.c:306  Error: add procs FAILED rc=-2
[jazz04:185068] spml_ucx.c:264  Error: ucp_ep_create(proc=131/224) failed: Destination is unreachable
[jazz04:185068] spml_ucx.c:306  Error: add procs FAILED rc=-2
[1555716468.435202] [jazz04:185068:0]           sock.c:116  UCX  ERROR connect(fd=217, dest_addr=2.1.3.9:45654) failed: Connection timed out
[1555716468.435197] [jazz04:185051:0]           sock.c:116  UCX  ERROR connect(fd=223, dest_addr=2.1.3.6:47881) failed: Connection timed out
[1555716468.435197] [jazz04:185059:0]           sock.c:116  UCX  ERROR connect(fd=225, dest_addr=2.1.3.9:53466) failed: Connection timed out
[1555716468.435197] [jazz04:185061:0]           sock.c:116  UCX  ERROR connect(fd=228, dest_addr=2.1.3.24:45613) failed: Connection timed out[jazz04:185067] spml_ucx.c:264  Error: ucp_ep_create(proc=141/224) failed: Destination is unreachable
[jazz04:185067] spml_ucx.c:306  Error: add procs FAILED rc=-2
[1555716468.436193] [jazz04:185067:0]           sock.c:116  UCX  ERROR connect(fd=227, dest_addr=2.1.3.21:57746) failed: Connection timed out--------------------------------------------------------------------------
It looks like SHMEM_INIT failed for some reason; your parallel process is
likely to abort.  There are many reasons that a parallel process can
fail during SHMEM_INIT; some of which are due to configuration or environment
problems.  This failure appears to be an internal failure; here's some
additional information (which may only be relevant to an Open SHMEM
developer):

  SPML add procs failed
  --> Returned "Out of resource" (-2) instead of "Success" (0)
--------------------------------------------------------------------------[jazz04:185059] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting
[jazz04:185061] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting
[jazz04:185067] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting[jazz04:185051] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting
[jazz04:185068] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting
[jazz09:77190] spml_ucx.c:264  Error: ucp_ep_create(proc=142/224) failed: Destination is unreachable
[jazz09:77190] spml_ucx.c:306  Error: add procs FAILED rc=-2[jazz09:77193] spml_ucx.c:264  Error: ucp_ep_create(proc=126/224) failed: Destination is unreachable
[jazz09:77193] spml_ucx.c:306  Error: add procs FAILED rc=-2[jazz09:77188] spml_ucx.c:264  Error: ucp_ep_create(proc=126/224) failed: Destination is unreachable
[jazz09:77188] spml_ucx.c:306  Error: add procs FAILED rc=-2[jazz09:77205] spml_ucx.c:264  Error: ucp_ep_create(proc=147/224) failed: Destination is unreachable
[jazz09:77205] spml_ucx.c:306  Error: add procs FAILED rc=-2
[jazz09:77185] spml_ucx.c:264  Error: ucp_ep_create(proc=138/224) failed: Destination is unreachable
[jazz09:77185] spml_ucx.c:306  Error: add procs FAILED rc=-2
[jazz09:77164] spml_ucx.c:264  Error: ucp_ep_create(proc=136/224) failed: Destination is unreachable
[jazz09:77164] spml_ucx.c:306  Error: add procs FAILED rc=-2
[jazz09:77154] spml_ucx.c:264  Error: ucp_ep_create(proc=133/224) failed: Destination is unreachable
[jazz09:77154] spml_ucx.c:306  Error: add procs FAILED rc=-2
[jazz09:77163] spml_ucx.c:264  Error: ucp_ep_create(proc=143/224) failed: Destination is unreachable
[jazz09:77163] spml_ucx.c:306  Error: add procs FAILED rc=-2
[jazz09:77160] spml_ucx.c:264  Error: ucp_ep_create(proc=162/224) failed: Destination is unreachable
[jazz09:77160] spml_ucx.c:306  Error: add procs FAILED rc=-2[jazz09:77156] spml_ucx.c:264  Error: ucp_ep_create(proc=150/224) failed: Destination is unreachable
[jazz09:77156] spml_ucx.c:306  Error: add procs FAILED rc=-2
[jazz09:77161] spml_ucx.c:264  Error: ucp_ep_create(proc=162/224) failed: Destination is unreachable
[jazz09:77161] spml_ucx.c:306  Error: add procs FAILED rc=-2
[1555716468.444513] [jazz09:77161:0]           sock.c:116  UCX  ERROR connect(fd=248, dest_addr=2.1.3.21:54105) failed: Connection timed out
[1555716468.444506] [jazz09:77164:0]           sock.c:116  UCX  ERROR connect(fd=222, dest_addr=2.1.3.9:41405) failed: Connection timed out
[1555716468.444503] [jazz09:77188:0]           sock.c:116  UCX  ERROR connect(fd=212, dest_addr=2.1.3.6:53664) failed: Connection timed out
[1555716468.444504] [jazz09:77193:0]           sock.c:116  UCX  ERROR connect(fd=212, dest_addr=2.1.3.6:39139) failed: Connection timed out
[1555716468.444509] [jazz09:77185:0]           sock.c:116  UCX  ERROR connect(fd=224, dest_addr=2.1.3.21:35155) failed: Connection timed out
[1555716468.444503] [jazz09:77190:0]           sock.c:116  UCX  ERROR connect(fd=228, dest_addr=2.1.3.6:41027) failed: Connection timed out
[1555716468.444506] [jazz09:77154:0]           sock.c:116  UCX  ERROR connect(fd=219, dest_addr=2.1.3.4:44068) failed: Connection timed out[1555716468.444503] [jazz09:77205:0]           sock.c:116  UCX  ERROR connect(fd=233, dest_addr=2.1.3.24:54378) failed: Connection timed out
[1555716468.444506] [jazz09:77163:0]           sock.c:116  UCX  ERROR connect(fd=229, dest_addr=2.1.3.8:41094) failed: Connection timed out
[1555716468.444512] [jazz09:77156:0]           sock.c:116  UCX  ERROR connect(fd=236, dest_addr=2.1.3.6:57853) failed: Connection timed out
[1555716468.444506] [jazz09:77160:0]           sock.c:116  UCX  ERROR connect(fd=248, dest_addr=2.1.3.21:36189) failed: Connection timed out[jazz09:77156] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting
[jazz09:77154] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting
[jazz09:77188] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting
[jazz09:77164] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting
[jazz09:77163] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting[jazz09:77193] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting
[jazz09:77190] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting[jazz09:77160] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting
[jazz09:77185] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting
[jazz09:77161] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting
[jazz09:77205] Error: pshmem_init.c:80 - _shmem_init() SHMEM failed to initialize - aborting
[jazz13:08994] spml_ucx.c:264  Error: ucp_ep_create(proc=124/224) failed: Destination is unreachable
[jazz13:08994] spml_ucx.c:306  Error: add procs FAILED rc=-2[1555716468.461250] [jazz13:8994 :0]           sock.c:116  UCX  ERROR connect(fd=210, dest_addr=2.1.3.4:56653) failed: Connection timed out
--------------------------------------------------------------------------
SHMEM_ABORT was invoked on rank 88 (pid 185059, host=jazz04) with errorcode -1.
```

Also in MPI tests. Cmd:
`mpirun -np 560 --display-map -mca btl self --tag-output --timestamp-output -mca pml ucx -mca coll '^hcoll' --bind-to core -x UCX_NET_DEVICES=ens1f0 -mca osc ucx -x UCX_IB_REG_METHODS=rcache,direct -x UCX_TLS=tcp -mca pmix_base_async_modex 1 -mca mpi_add_procs_cutoff 0 -mca pmix_base_collect_data 0 --map-by node /hpc/local/benchmarks/hpcx_install_2019-04-20/mtt-tests-gcc/installs/CWmU/tests/mpich_tests/mtt-tests.git/mpich/test/mpi/coll/icalltoallv`

Output:
```sh
Sat Apr 20 22:59:55 2019[1,220]<stderr>:[clx-orion-037:24850] pml_ucx.c:380  Error: ucp_ep_create(proc=219) failed: Destination is unreachable
Sat Apr 20 22:59:55 2019[1,220]<stderr>:[clx-orion-037:24850] pml_ucx.c:447  Error: Failed to resolve UCX endpoint for rank 219
Sat Apr 20 22:59:55 2019[1,220]<stdout>:[1555790395.783035] [clx-orion-037:24850:0]         select.c:410  UCX  ERROR no active messages transport to <no debug data>: tcp/ens1f0 - Destination is unreachableSat Apr 20 22:59:55 2019[1,160]<stderr>:[clx-orion-037:24847] pml_ucx.c:380  Error: ucp_ep_create(proc=159) failed: Destination is unreachable
Sat Apr 20 22:59:55 2019[1,160]<stderr>:[clx-orion-037:24847] pml_ucx.c:447  Error: Failed to resolve UCX endpoint for rank 159
Sat Apr 20 22:59:55 2019[1,160]<stdout>:[1555790395.783572] [clx-orion-037:24847:0]         select.c:410  UCX  ERROR no active messages transport to <no debug data>: tcp/ens1f0 - Destination is unreachableSat Apr 20 22:59:55 2019[1,60]<stderr>:[clx-orion-037:24842] pml_ucx.c:380  Error: ucp_ep_create(proc=59) failed: Destination is unreachable
Sat Apr 20 22:59:55 2019[1,60]<stderr>:[clx-orion-037:24842] pml_ucx.c:447  Error: Failed to resolve UCX endpoint for rank 59Sat Apr 20 22:59:55 2019[1,60]<stdout>:[1555790395.784118] [clx-orion-037:24842:0]         select.c:410  UCX  ERROR no active messages transport to <no debug data>: tcp/ens1f0 - Destination is unreachable
Sat Apr 20 22:59:55 2019[1,540]<stderr>:[clx-orion-037:24879] pml_ucx.c:380  Error: ucp_ep_create(proc=539) failed: Destination is unreachable
Sat Apr 20 22:59:55 2019[1,540]<stderr>:[clx-orion-037:24879] pml_ucx.c:447  Error: Failed to resolve UCX endpoint for rank 539Sat Apr 20 22:59:55 2019[1,540]<stdout>:[1555790395.784621] [clx-orion-037:24879:0]         select.c:410  UCX  ERROR no active messages transport to <no debug data>: tcp/ens1f0 - Destination is unreachable
Sat Apr 20 22:59:55 2019[1,120]<stderr>:[clx-orion-037:24845] pml_ucx.c:380  Error: ucp_ep_create(proc=119) failed: Destination is unreachable
Sat Apr 20 22:59:55 2019[1,120]<stderr>:[clx-orion-037:24845] pml_ucx.c:447  Error: Failed to resolve UCX endpoint for rank 119Sat Apr 20 22:59:55 2019[1,120]<stdout>:[1555790395.785157] [clx-orion-037:24845:0]         select.c:410  UCX  ERROR no active messages transport to <no debug data>: tcp/ens1f0 - Destination is unreachable
[clx-orion-037:24850] *** An error occurred in MPI_Comm_split
[clx-orion-037:24850] *** reported by process [2221473793,220]
[clx-orion-037:24850] *** on communicator MPI_COMM_WORLD
[clx-orion-037:24850] *** MPI_ERR_OTHER: known error not in list
[clx-orion-037:24850] *** MPI_ERRORS_ARE_FATAL (processes in this communicator will now abort,
```

Configuration
```
OMPI: 4.0.2a1
MOFED: MLNX_OFED_LINUX-4.5-1.0.1.0
Module: hpcx-gcc (2019-04-20)
Test module: mtt-tests/hpcx-gcc
Nodes: orion x20 (ppn=28(x20), nodelist=clx-orion-[037-047,049,051-057,093])
Job: ucx-roce
```

MTT log: http://e2e-gw.mellanox.com:4080/hpc/scrap/users/mtt/scratch/ucx_ompi/20190420_225555_28287_352698_clx-orion-037/html/test_stdout_B9ZBz_.txt

Reproduced on master:
```
OMPI: 4.0.2a1
MOFED: MLNX_OFED_LINUX-4.5-1.0.1.0
Module: hpcx-gcc (2019-06-26)
Test module: mtt-tests/hpcx-gcc
Nodes: orion x13 (ppn=28(x13), nodelist=clx-orion-[058-064,097-102])
```

MTT log: http://e2e-gw.mellanox.com:4080/mnt/lustre/users/mtt/scratch/ucx_ompi/20190626_223249_26692_401512_clx-orion-058/html/test_stdout_QRFS4X.txt

Cmd:
`mpirun -np 364 --display-map -mca btl self --tag-output --timestamp-output -mca pml ucx -mca coll '^hcoll' --bind-to core -x UCX_NET_DEVICES=ens1f0 -mca osc ucx -x UCX_IB_REG_METHODS=rcache,direct -x UCX_TLS=tcp -mca pmix_base_async_modex 1 -mca mpi_add_procs_cutoff 0 -mca pmix_base_collect_data 0 --map-by node /hpc/local/benchmarks/hpcx_install_2019-06-26/mtt-tests-gcc/installs/uNmj/tests/mpich_tests/mtt-tests.git/mpich/test/mpi/coll/allgather3`

Output:
```sh
Wed Jun 26 22:35:41 2019[1,273]<stderr>:[clx-orion-058:21759] pml_ucx.c:380  Error: ucp_ep_create(proc=272) failed: Destination is unreachable
Wed Jun 26 22:35:41 2019[1,273]<stderr>:[clx-orion-058:21759] pml_ucx.c:447  Error: Failed to resolve UCX endpoint for rank 272
Wed Jun 26 22:35:41 2019[1,273]<stdout>:[1561577741.153557] [clx-orion-058:21759:0]         select.c:401  UCX  ERROR no active messages transport to <no debug data>: tcp/ens1f0 - Destination is unreachableWed Jun 26 22:35:41 2019[1,299]<stderr>:[clx-orion-058:21763] pml_ucx.c:380  Error: ucp_ep_create(proc=298) failed: Destination is unreachable
Wed Jun 26 22:35:41 2019[1,299]<stderr>:[clx-orion-058:21763] pml_ucx.c:447  Error: Failed to resolve UCX endpoint for rank 298
Wed Jun 26 22:35:41 2019[1,299]<stdout>:[1561577741.153924] [clx-orion-058:21763:0]         select.c:401  UCX  ERROR no active messages transport to <no debug data>: tcp/ens1f0 - Destination is unreachable
Wed Jun 26 22:35:41 2019[1,351]<stderr>:[clx-orion-058:21771] pml_ucx.c:380  Error: ucp_ep_create(proc=350) failed: Destination is unreachable
Wed Jun 26 22:35:41 2019[1,351]<stderr>:[clx-orion-058:21771] pml_ucx.c:447  Error: Failed to resolve UCX endpoint for rank 350
Wed Jun 26 22:35:41 2019[1,351]<stdout>:[1561577741.154241] [clx-orion-058:21771:0]         select.c:401  UCX  ERROR no active messages transport to <no debug data>: tcp/ens1f0 - Destination is unreachableWed Jun 26 22:35:41 2019[1,247]<stderr>:[clx-orion-058:21755] pml_ucx.c:380  Error: ucp_ep_create(proc=246) failed: Destination is unreachable
Wed Jun 26 22:35:41 2019[1,247]<stderr>:[clx-orion-058:21755] pml_ucx.c:447  Error: Failed to resolve UCX endpoint for rank 246
Wed Jun 26 22:35:41 2019[1,247]<stdout>:[1561577741.154643] [clx-orion-058:21755:0]         select.c:401  UCX  ERROR no active messages transport to <no debug data>: tcp/ens1f0 - Destination is unreachableWed Jun 26 22:35:41 2019[1,325]<stderr>:[clx-orion-058:21767] pml_ucx.c:380  Error: ucp_ep_create(proc=324) failed: Destination is unreachable
Wed Jun 26 22:35:41 2019[1,325]<stderr>:[clx-orion-058:21767] pml_ucx.c:447  Error: Failed to resolve UCX endpoint for rank 324Wed Jun 26 22:35:41 2019[1,325]<stdout>:[1561577741.154947] [clx-orion-058:21767:0]         select.c:401  UCX  ERROR no active messages transport to <no debug data>: tcp/ens1f0 - Destination is unreachable
Wed Jun 26 22:35:41 2019[1,221]<stderr>:[clx-orion-058:21751] pml_ucx.c:380  Error: ucp_ep_create(proc=220) failed: Destination is unreachable
Wed Jun 26 22:35:41 2019[1,221]<stderr>:[clx-orion-058:21751] pml_ucx.c:447  Error: Failed to resolve UCX endpoint for rank 220
Wed Jun 26 22:35:41 2019[1,221]<stdout>:[1561577741.155438] [clx-orion-058:21751:0]         select.c:401  UCX  ERROR no active messages transport to <no debug data>: tcp/ens1f0 - Destination is unreachableWed Jun 26 22:35:41 2019[1,90]<stderr>:[clx-orion-102:27990] pml_ucx.c:380  Error: ucp_ep_create(proc=91) failed: Destination is unreachable
Wed Jun 26 22:35:41 2019[1,90]<stderr>:[clx-orion-102:27990] pml_ucx.c:447  Error: Failed to resolve UCX endpoint for rank 91Wed Jun 26 22:35:41 2019[1,90]<stdout>:[1561577741.148019] [clx-orion-102:27990:0]         select.c:401  UCX  ERROR no active messages transport to <no debug data>: tcp/ens1f0 - Destination is unreachable
Wed Jun 26 22:35:41 2019[1,38]<stderr>:[clx-orion-102:27986] pml_ucx.c:380  Error: ucp_ep_create(proc=39) failed: Destination is unreachable
Wed Jun 26 22:35:41 2019[1,38]<stderr>:[clx-orion-102:27986] pml_ucx.c:447  Error: Failed to resolve UCX endpoint for rank 39[clx-orion-058:21767] *** An error occurred in MPI_Allreduce
[clx-orion-058:21767] *** reported by process [2700673025,325]
[clx-orion-058:21767] *** on communicator MPI_COMM_WORLD
[clx-orion-058:21767] *** MPI_ERR_OTHER: known error not in list
[clx-orion-058:21767] *** MPI_ERRORS_ARE_FATAL (processes in this communicator will now abort,
[clx-orion-058:21767] ***    and potentially your MPI job)Wed Jun 26 22:35:41 2019[1,38]<stdout>:[1561577741.148290] [clx-orion-102:27986:0]         select.c:401  UCX  ERROR no active messages transport to <no debug data>: tcp/ens1f0 - Destination is unreachable
Wed Jun 26 22:35:41 2019[1,194]<stderr>:[clx-orion-102:27998] pml_ucx.c:380  Error: ucp_ep_create(proc=195) failed: Destination is unreachable
Wed Jun 26 22:35:41 2019[1,194]<stderr>:[clx-orion-102:27998] pml_ucx.c:447  Error: Failed to resolve UCX endpoint for rank 195Wed Jun 26 22:35:41 2019[1,194]<stdout>:[1561577741.148899] [clx-orion-102:27998:0]         select.c:401  UCX  ERROR no active messages transport to <no debug data>: tcp/ens1f0 - Destination is unreachable
Wed Jun 26 22:35:41 2019[1,116]<stderr>:[clx-orion-102:27992] pml_ucx.c:380  Error: ucp_ep_create(proc=117) failed: Destination is unreachable
Wed Jun 26 22:35:41 2019[1,116]<stderr>:[clx-orion-102:27992] pml_ucx.c:447  Error: Failed to resolve UCX endpoint for rank 117[clx-orion-058:21694] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2079
[clx-orion-058:21694] PMIX ERROR: UNREACHABLE in file server/pmix_server.c at line 2079
```

Devices status before and after the test is OK:
```sh
----------------
clx-orion-[058-064,097-102]
----------------
ens1f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
```

trying to run application usin openmpi 2.0.2 and ucx 1.6 in the following way:

mpirun -v -np 2000 -npernode 40 -mca pml ucx -x UCX_NET_DEVICES=mlx5_0:1 -x PATH -x LD_LIBRARY_PATH -x OMP_NUM_THREADS=1 -machinefile $PBS_NODEFILE ./a.out

error is:
select.c:410  UCX  ERROR no active messages transport to <no debug data>: mm/posix - Destination is unreachable, mm/sysv - Destination is unreachable, self/self - Destination is unreachable, rc/mlx5_0:1 - Destination is unreachable, rc_mlx5/mlx5_0:1 - Destination is unreachable, dc_mlx5/mlx5_0:1 - Destination is unreachable,

pml_ucx.c:256 Error: Failed to connect to proc: 256, Destination is unreachable


Any suggestions!

--- a/src/uct/tcp/tcp_iface.c
+++ b/src/uct/tcp/tcp_iface.c
@@ -91,12 +91,9 @@ static int uct_tcp_iface_is_reachable(const uct_iface_h tl_iface,
                                       const uct_device_addr_t *dev_addr,
                                       const uct_iface_addr_t *iface_addr)
 {
-    uct_tcp_iface_t *iface         = ucs_derived_of(tl_iface, uct_tcp_iface_t);
-    const in_addr_t *remote_inaddr = (const in_addr_t*)dev_addr;
-    in_addr_t netmask              = iface->config.netmask.sin_addr.s_addr;
-
-    return (*remote_inaddr & netmask) ==
-           (iface->config.ifaddr.sin_addr.s_addr & netmask);
+    /* We always report that a peer is reachable. connect() call will
+     * fail if the peer is unreachable when creating UCT/TCP EP */
+    return 1;
 }
 
 static ucs_status_t uct_tcp_iface_query(uct_iface_h tl_iface, uct_iface_attr_t *attr)

